pages/account.js

```diff
@@ -22,6 +22,8 @@ class Account extends Component {
             isAuthenticated: false,
             name: '',
             email: '',
```

Comment:
let's call this state `oldEmail` and `oldName` i think it will be less confusing